### DISCUSSION TITLE
Routes Walletkit

### DISCUSSION THREAD

**Discussion by oriana13** [walt-id/.github#45] [Help] [answered]
Hello Waltid team !

How to access this sskit route: /v1/credentials/issue from the walletkit ?
Can you explain to me how to make this route work from the walletkit deployment please?

Thank you, I wish you a good day

**Comment by taminobaumann** [comment, marked as answer]
Hey,
Thanks for reaching out!
The SSI-Kit endpoint you are mentioning is not exposed in the wallet-kit. The wallet-kit is using the [OIDC4VC](https://openid.net/specs/openid-4-verifiable-credential-issuance-1_0.html) flow to issue credentials.

One example on how to start the issuance of a credential to a wallet would be to use this endpoint:

POST /issuer-api/{tenantId}/credentials/issuance/request?isPreAuthorized=true&walletId=$walletId

Body Schema:
{
    "credentials": [
        {
        "credentialData": {
            "my data": "..."
        },
        "type": "string"
        }
    ]
}

Body Example:
{
    "credentials": [
        {
        "credentialData": {
            "credentialSubject": {
                "currentAddress":["1 Boulevard de la Liberté, 59800 Lille"],"dateOfBirth":"1993-04-08","familyName":"DOE","firstName":"Jane","gender":"FEMALE","id":"did:ebsi:2AEMAqXWKYMu1JHPAgGcga4dxu7ThgfgN95VyJBJGZbSJUtp","nameAndFamilyNameAtBirth":"Jane DOE","personalIdentifier":"0904008084H","placeOfBirth":"LILLE, FRANCE"}
        },
        "type": "VerifiableId"
        }
    ]
}
